File: app/src/main/java/com/github/alexeygorovoy/moxytemplate/api/HeroApi.java
package com.github.alexeygorovoy.moxytemplate.api;


import com.github.alexeygorovoy.moxytemplate.api.models.Heroes;

import retrofit2.http.GET;
import rx.Observable;


public interface HeroApi {

    @GET("test-mobile/iOS/json/test2.json")
    Observable<Heroes> getHeroes();

}

File: app/src/main/java/com/github/alexeygorovoy/moxytemplate/api/HeroApi.kt
package com.github.alexeygorovoy.moxytemplate.api

import com.github.alexeygorovoy.moxytemplate.api.models.Heroes
import io.reactivex.Observable

import retrofit2.http.GET

interface HeroApi {

    @GET("test-mobile/iOS/json/test2.json")
    fun getHeroes(): Observable<Heroes>
}
